charts/chainlink-cluster/devspace.yaml

```diff
@@ -25,47 +25,14 @@ pipelines:
       start_dev app                # 3. Start dev mode "app" (see "dev" section)
   deploy:
     run: |-
-      set -o pipefail
-      echo "Removing .devspace cache!"
-      rm -rf .devspace/ || true
-      registry_id=$(echo "$DEVSPACE_IMAGE" | cut -d'.' -f1)
-
-      # Login into registry
-      echo "Authorizing into ECR registry"
-      aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin ${registry_id}.dkr.ecr.us-west-2.amazonaws.com
```

Comment:
I like the simplification here. Would we just need to document that users should do the ECR auth step outside of devspace? To me this sounds reasonable.

Reply:
Yeah, the devspace documentation expects you login first https://www.devspace.sh/docs/configuration/images/push#authentication

Reply:
This is reasonable but I just added it here to remove another thing devs need to worry about, up to you, I prefer my builds to always work. It's such a pain to re-authorize several times a day, fix not working VPN, or do another ECR authorization. @chainchad @HenryNguyen5

Reply:
id rather keep it out, and if we really need it, have it as a wrapper around devspace. my reasoning is that i'd like to stay true to devspace docs as much as i can

Reply:
I'm in favor of keeping it out. Authing is something I have to do anyway so there's no point in re-authing here.